hudi-client/hudi-client-common/src/main/java/org/apache/hudi/execution/HoodieLazyInsertIterable.java

```diff
@@ -90,12 +90,16 @@ public R getResult() {
     }
   }
 
-  static <T> Function<HoodieRecord<T>, HoodieInsertValueGenResult<HoodieRecord>> getTransformer(Schema schema,
+  /**
+   * Transformer function to help transform a HoodieRecord. This transformer is used by BufferedIterator to offload some
+   * expensive operations of transformation to the reader thread.
+   */
+  public <T> Function<HoodieRecord<T>, HoodieInsertValueGenResult<HoodieRecord>> getTransformer(Schema schema,
                                                                                                 HoodieWriteConfig writeConfig) {
```

Comment:
Do we still need this method? Didn't see any override from it.

Reply:
In the runtime, we are providing a class, that transforms the record. The logic resides in other repository, so, we need this method to be extensible.